content/applications/finance/accounting/taxes/vat_validation.rst

```diff
@@ -1,43 +1,44 @@
-===========================
-VIES VAT numbers validation
-===========================
+===============================
+VAT numbers verification (VIES)
+===============================
 
-**VAT Information Exchange System** - abbreviated **VIES** - is a tool provided by the European
-Commission that allows you to check the validity of VAT numbers of companies registered in the
-European Union.
+`VAT Information Exchange System <https://ec.europa.eu/taxation_customs/vies/#/vat-validation>`_, or
+**VIES**, is a tool provided by the European Commission that allows you to check the validity of VAT
+numbers for companies registered in the European Union.
 
-Odoo provides a feature to **Verify VAT Numbers** when you save a contact. This helps you make sure
-that your contacts provided you with a valid VAT number without leaving Odoo interface.
+Odoo uses this tool to verify your contacts' VAT numbers directly from Odoo's interface.
```

Comment:
Nice to start talking about odoo but to rephrase slightly with something like:
Odoo allows verifying  ....